manifests/cluster-network-addons/0.12.0/cluster-network-addons-operator.0.12.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,201 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: cluster-network-addons-operator.0.12.0
+  namespace: placeholder
```

Comment:
What is the "placeholder" namespace?
I don't see it being replaced in any script. Where do we replace it?
It's been here since always.

Reply:
OLM replaces it by the namespace provided by administrator installing given component.